clang/lib/Sema/Sema.cpp

```diff
@@ -1594,6 +1596,7 @@ void Sema::markKnownEmitted(
     assert(!IsKnownEmitted(S, C.Callee) &&
            "Worklist should not contain known-emitted functions.");
     S.DeviceKnownEmittedFns[C.Callee] = {C.Caller, C.Loc};
+
```

Comment:
This is not related change. Please revert.
```suggestion
```

Reply:
Done